### ISSUE TITLE
Error: Type 'Uint8List' not found

### ISSUE THREAD

**Issue by KeithDC-SD** [peiffer-innovations/automated_testing_framework#27]
Hi there,

We don't explicitly use this package in our app, but we recently began experiencing test failures stemming from a recent upgrade, as this package appears to be a transitive dependency on another package. 

To circumvent this we've explicitly declared the previous version of this package which appears to have solved the testing errors (`automated_testing_framework: 4.0.7+1`).

The errors appear to mostly stem from:
```
/root/.pub-cache/hosted/pub.dartlang.org/automated_testing_framework-4.0.7+2/lib/src/driver/handlers/screenshot_handler.dart:12:3: Error: Type 'Uint8List' not found.
  Uint8List? _screenshot;
```

Thoughts?

**Comment by KeithDC-SD** [comment]
Update: We just found that the dependency is `json_dynamic_widget`, but because we do explicit version control, we're going to see if upgrading that package helps (most likely will, and if so, may be a lesson for us using static versioning, or something to consider anyways).

**Comment by KeithDC-SD** [comment]
**Solved**: Upgraded to `json_dynamic_widget: 5.1.2+4`.
